libs/symphonia/src/bin/symphtest.rs

```diff
@@ -1,65 +1,197 @@
-
-    #[structopt(
-        name = "variance",
-        long,
-        short,
-        help = "Variance of normal distribution for latency",
-        default_value = "20.0"
-    )]
-    latency_variance: f64,
-
-    #[structopt(
-        name = "loss",
-        long,
-        short,
-        help = "Probability of loss per network transfer",
-        default_value = "0.05"
-    )]
-    loss_prob: f64,
-
-    #[structopt(
-        name = "weights",
-        long,
-        short,
-        help = "Comma separated voting weight of each consensus participants",
-        default_value = "100",
-        use_delimiter = true
-    )]
-    participant_weights: Vec<u64>,
+enum Opt {
+    #[structopt(about = "Simulate rounds in Symphonia for a set of params")]
+    Rounds {
+        #[structopt(
+            name = "rounds",
+            long,
+            short,
+            help = "Number of rounds or times to reach consensus on a block",
+            default_value = "1"
+        )]
+        rounds: u64,
+
+        #[structopt(
+            name = "mean",
+            long,
+            short,
+            help = "Mean time in ms for latency",
+            default_value = "100.0"
+        )]
+        latency_mean_ms: f64,
+
+        #[structopt(
+            name = "deviation",
+            long,
+            short,
+            help = "Standard deviation of normal distribution for latency",
+            default_value = "5.0"
+        )]
+        latency_standard_deviation: f64,
+
+        #[structopt(
+            name = "loss",
+            long,
+            short,
+            help = "Probability of loss per network transfer",
+            default_value = "0.05"
+        )]
+        loss_prob: f64,
+
+        #[structopt(
+            name = "weights",
+            long,
+            short,
+            help = "Comma separated voting weight of each consensus participants",
+            default_value = "100",
+            use_delimiter = true
+        )]
+        participant_weights: Vec<u64>,
+    },
+
+    #[structopt(about = "Simulate different test cases selection params from input file")]
+    Parameterize {
+        #[structopt(
+            name = "test-count",
+            long,
+            short,
+            help = "Number of test simulations to run",
+            default_value = "1"
+        )]
+        test_count: u64,
+
+        #[structopt(
+            name = "rounds",
+            long,
+            short,
+            help = "Number of rounds or times to reach consensus on a block",
+            default_value = "1"
+        )]
+        rounds: u64,
+
+        #[structopt(
+            name = "filename",
+            long,
+            short,
+            help = "Input params file name containing to determine values to test"
+        )]
+        file_name: String,
+    },
+}
+
+#[derive(Debug, Deserialize)]
+struct NetworkLatencyParams {
+    mean_milli_sec: Vec<f64>,
+    standard_deviation: Vec<f64>,
+    loss_probability: Vec<f64>,
```

Comment:
There wasn't a simple way to import a tuple and using toml tables implies they'd be hashmaps so decided to just use vectors.